pyproject.toml

```diff
@@ -8,7 +8,7 @@ version = "1.2.0"
 description = "Django adapter for the InertiaJS framework"
 authors = [{name="Brandon Shar", email="brandon@bellawatt.com"}]
 license = "MIT"
-requires-python = ">=3.9"
+requires-python = ">=3.10"
```

Comment:
I'm always for using more modern versions, but this is an important change that perhaps should be done elsewhere?

If it is because of typing features, why it would be best to just use `from __future__ import annotations` to enable lazy evaluations of type hints. Then as long as the type checker is run with a modern version of Python it will be fine.

Reply:
That's a good call. django-stubs requires Python 3.10, so I could not add the dependency bumping the Python version. As Python 3.9 is reaching EOL on October, I thought it was harmless to do that, but I agree it should be done as another PR. I'll open it later today.

Reply:
I suppose that is coming up very soon. I'm all for dropping 3.9, it just feels like something that should be dropped in a bigger release. Awesome, thanks!